x/stakeibc/keeper/icacallbacks_delegate.go

```diff
@@ -56,8 +56,6 @@ func DelegateCallback(k Keeper, ctx sdk.Context, packet channeltypes.Packet, ack
 
 	if ack == nil {
 		// timeout
-		depositRecord.Status = recordstypes.DepositRecord_DELEGATION_QUEUE
```

Comment:
Do we not need to handle this logic here, because a timeout will always close the channel, which would trigger `restore_interchain_account`?

Reply:
Yep!